include/ign-rviz/VisualizationManager.h

```diff
@@ -33,29 +34,29 @@
 
 class VisualizationManager : public SceneManager {
  private:
-  ros::Subscriber point_subscriber;
-  ros::Subscriber pose_subscriber;
-  ros::Subscriber orientation_subscriber;
-  ros::Subscriber marker_subscriber;
-  ros::Subscriber tf_subscriber;
-  ros::Subscriber pointcloud_subscriber;
-  ros::NodeHandle nh;
+  rclcpp::Subscription<geometry_msgs::msg::PointStamped>::SharedPtr point_subscriber;
+  rclcpp::Subscription<geometry_msgs::msg::PoseStamped>::SharedPtr pose_subscriber;
+  rclcpp::Subscription<sensor_msgs::msg::Imu>::SharedPtr orientation_subscriber;
+  rclcpp::Subscription<visualization_msgs::msg::Marker>::SharedPtr marker_subscriber;
+  rclcpp::Subscription<tf2_msgs::msg::TFMessage>::SharedPtr tf_subscriber;
+  rclcpp::Subscription<sensor_msgs::msg::PointCloud2>::SharedPtr pointcloud_subscriber;
+  rclcpp::Node::SharedPtr nh;
   AxisVisualPtr axis;
   MarkerPtr pcl_marker;
 
-  tf2_ros::Buffer tfBuffer;
-  tf2_ros::TransformListener tfListener;
+  std::shared_ptr<tf2_ros::Buffer> tfBuffer;
+  std::shared_ptr<tf2_ros::TransformListener> tfListener;
 
  public:
-  VisualizationManager(int &argc, char** argv);
-  void point_callback(const geometry_msgs::PointStampedConstPtr&);
```

Comment:
can you add docblock? here an example https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/clock.hpp#L58

Reply:
Sure